src/modules/Layout/Nav/Nav.scss

```diff
@@ -0,0 +1,20 @@
+@import 'index.scss';
+
+.j-nav {
+  display: flex; 
+  height: $hf-height-header;
+
+  &__item {
+    display: flex;
+    height: 100%;
+    align-items: center;
+    padding: 0 $hf-padding-middle;
+    color: $hf-color-white;
+    text-decoration: none;
+
+    &.active {
```

Comment:
`.active` is added by `react-router` **v4** , that's why in this particular case we don't use BEM--modificator stuff.